libs/dh/admin/feature-user-management/tsconfig.lib.json

```
@@ -13,5 +13,11 @@
     "jest.config.ts",
     "**/*.test.ts"
   ],
-  "include": ["**/*.ts"]
+  "include": [
+    "**/*.ts",
+    "../feature-user-roles/src/lib/drawer/dh-role-drawer.component.ts",
+    "../shared/src/dh-role-status.component.ts",
+    "../shared/src/dh-user-status.component.ts",
+    "../shared/src/dh-tab-data-general-error.component.ts"
+  ]
```

suggestion: this should probably be reverted. The files are in another lib.

```suggestion
  "include": ["**/*.ts"]
```